railties/lib/rails/application/finisher.rb

```diff
@@ -173,6 +173,22 @@ def self.complete(_state)
         end
       end
 
+      initializer :add_builtin_route do |app|
+        if Rails.env.development?
+          app.routes.prepend do
+            get "/rails/info/properties" => "rails/info#properties", internal: true
+            get "/rails/info/routes"     => "rails/info#routes", internal: true
+            get "/rails/info"            => "rails/info#index", internal: true
+          end
+
+          routes_reloader.after_load_paths_callback = Proc.new do
```

Comment:
```suggestion
          routes_reloader.after_load_paths = -> do
```

We generally prefer the lambda syntax for these types of invocations.